apps/vor/frontend/src/components/PagePickGuardian/PagePickGuardian.tsx

```diff
@@ -145,7 +146,9 @@ export const PagePickGuardian: FC = () => {
             }}
           >
             <Icon as={PlusIcon} mt="5px" mr={3} fill="primary" />
-            <Typography.Body>Create {name || "new guardian"}</Typography.Body>
+            <Typography.Body>
+              <Trans>Create</Trans> {name || t`new guardian`}
```

Comment:
this should be:
```suggestion
              <Trans>Create</Trans> {name || i18n._(t`new guardian`)}
```
else "new guardian" won't be translated. You could get the i18n object by using the useLingui hook. `const {i18n} = useLingui()`

Reply:
also, run yarn test -u to update the snapshots.